test/run_test

```diff
@@ -16,6 +16,7 @@ shopt -s nullglob
 . test/pg-test-lib.sh
 
 TEST_LIST="\
+ct_test_distgen_generated_files
```

Comment:
Very much doubt the test should be run over here, part of the regular test suite. How will this work when we run the test downstream?

Reply:
Good point. I have missed the downstream part. The test suite should not be a part of downstream. It does not make sense at all. Thank you.